templates/easy_breadcrumb/easy_breadcrumb.component.inc

```diff
@@ -15,6 +15,8 @@ function europa_atomium_theme_easy_breadcrumb() {
         'breadcrumb' => NULL,
         'segments_quantity' => NULL,
         'separator' => NULL,
+        'here' => NULL,
```

Comment:
Is it necessary to have the `here` variable ? Shouldn't we include this directly in the .tpl.php ?